src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -384,6 +384,11 @@
     <ProjectReference Include="..\..\System.Diagnostics.Debug\src\System.Diagnostics.Debug.csproj">
       <UndefineProperties>%(ProjectReference.UndefineProperties);TargetGroup</UndefineProperties>
     </ProjectReference>
+    <ProjectReference Include="..\..\System.Security.Principal\src\System.Security.Principal.csproj">
```

Comment:
I have added this to issue #12850 to track cleaning these up after stopping to expose those types from CoreLib.